### PR TITLE
bendale2015towards_1d draft

### PR DESCRIPTION
**Pull request by ywen666**

### CODE REVIEW

File: _summaries/bendale2015towards_1.md [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+<img width="400px" src="bendale2015towards_1a.png"/>  
+
+**Design**: “open space risk” should be measured in feature space, rather than in pixel space. The key question: “is there a feature space, ideally a layer in the deep network, where these adversarial images are far away from training examples.
+
+<img width="800px" src="bendale2015towards_1b.png"/>  
+
+* Multi-class Meta-Recognition
+  * Each class is represented as a point, a mean activation vector (MAV) with the mean computed over only the correctly classified training examples
+  * Given the MAV and an input image, the authors measure distance between them. They directly threshold distance to determine an overall maximum distance threshold.
+  * In particular, on line 3 of Alg. 1 we use the libMR FitHigh function to do Weibull fitting on the largest of the distances between all correct positive training instances and the associated $µ_i$. This results in a parameter $\rho_i$ , which is used to estimate the probability of an input being an outlier with respect to class i.
```
Comment (ayushchauhan) [inline _summaries/bendale2015towards_1.md:26]: 1. Here as well as elsewhere, you should enclose the math symbols inside double dollar signs ($$....$$)
2. Probably use j for indexing to be consistent with the algorithm?

File: _summaries/bendale2015towards_1.md [hunk 2]
```diff
@@ -1,19 +1,42 @@
...
+* Interpretation of Activation Vectors
+  * Closed Set: The authors hypothesize that for most categories, there is a relatively consistent pattern of related activations. The MAV captures that distribution as a single point. The AVs present a space where we measure the distance from an input image in terms of the activation of each class.
+  * Open Set: The activation vectors of the input with the MAV for a class for which the input produced maximum response, we observe it is often far from the mean. However, for some open set images the response provided is close to the AV but still has an overall low activation level.
```
Comment (ayushchauhan) [inline _summaries/bendale2015towards_1.md:29]: Include for fooling set and adversarial sets as well?

File: _summaries/bendale2015towards_1.md [hunk 3]
```diff
@@ -1,19 +1,42 @@
...
+* OpenMax Layer
+  * In open set prediction, it is not appropriate to require the probabilities to sum to 1.
+  * Let $\rho$ be a vector of metarecognition models for each class estimated by Alg. 1. Alg. 2 summarize the steps for OpenMax computation.
+  * The unknown unknown class to be at index 0.
+  * Use the Weibull CDF probability (line 3 of Alg. 2) on the distance between x and µi for the core of the rejection estimation. The model $\mu_i$ is computed using the images associated with category i, images that were classified correctly (top-1) during training process.
```
Comment (ayushchauhan) [inline _summaries/bendale2015towards_1.md:34]: Double dollar signs.

File: _summaries/bendale2015towards_1.md [hunk 4]
```diff
@@ -1,19 +1,42 @@
...
-* Bullets
-* To highlight the core concepts
+**Problem** It is easy to generate images that humans would never classify as a particular object class, yet networks classify such images high confidence as that given class. The closed set nature of deep networks forces them to choose from one of the known classes leading to such artifacts. However, recognition in the real world is open set, i.e. the recognition system should reject unknown/unseen classes at test time.
+
+The SoftMax layer is a significant component of the problem because of its closed nature. The authors propose an alternative, OpenMax, which extends SoftMax layer by enabling it to predict an unknown class.
+
+## OpenMax
+<img width="400px" src="bendale2015towards_1a.png"/>  
+
+**Design**: “open space risk” should be measured in feature space, rather than in pixel space. The key question: “is there a feature space, ideally a layer in the deep network, where these adversarial images are far away from training examples.
```
Comment (ayushchauhan) [inline _summaries/bendale2015towards_1.md:19]: End with ?"

File: _summaries/bendale2015towards_1.md [hunk 5]
```diff
@@ -1,19 +1,42 @@
...
+* Interpretation of Activation Vectors
+  * Closed Set: The authors hypothesize that for most categories, there is a relatively consistent pattern of related activations. The MAV captures that distribution as a single point. The AVs present a space where we measure the distance from an input image in terms of the activation of each class.
+  * Open Set: The activation vectors of the input with the MAV for a class for which the input produced maximum response, we observe it is often far from the mean. However, for some open set images the response provided is close to the AV but still has an overall low activation level.
+* OpenMax Layer
+  * In open set prediction, it is not appropriate to require the probabilities to sum to 1.
+  * Let $\rho$ be a vector of metarecognition models for each class estimated by Alg. 1. Alg. 2 summarize the steps for OpenMax computation.
+  * The unknown unknown class to be at index 0.
```
Comment (ayushchauhan) [inline _summaries/bendale2015towards_1.md:33]: Probably define the "unknown unknowns" class?

**Comment by ayushchauhan** [comment]
I liked the summary overall. Touches upon all the key points given the size constraint. It does rely a little bit on the speaker to explain some points.

File: _summaries/bendale2015towards_1.md (hunk 1 above)
Comment (ywen666) [reply _summaries/bendale2015towards_1.md:26]: done

File: _summaries/bendale2015towards_1.md (hunk 2 above)
Comment (ywen666) [reply _summaries/bendale2015towards_1.md:29]: done

File: _summaries/bendale2015towards_1.md (hunk 3 above)
Comment (ywen666) [reply _summaries/bendale2015towards_1.md:34]: done

File: _summaries/bendale2015towards_1.md (hunk 4 above)
Comment (ywen666) [reply _summaries/bendale2015towards_1.md:19]: done

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**